ci-operator/step-registry/openshift-extended/web-tests/openshift-extended-web-tests-ref.yaml

```diff
@@ -6,7 +6,7 @@ ref:
   timeout: 8h0m0s
   resources:
     requests:
-      cpu: "2"
+      cpu: "100m"
       memory: 2Gi
```

Comment:
Per the [thread](https://redhat-internal.slack.com/archives/C015H2WDJRY/p1698070785033039), seem like not only cpu hit bottleneck, also hit OOM, so shall we shrink the memory request to 600Mi?

To be safer, shall we set limits for `cpu`?

Another thing is there is `resources` setting in our ci job config level, e.g: openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml, I am not sure the job config level setting will overwrite the step setting?